docs/modules/reference/pages/daemons/daemon-config-files/event-translator.adoc

```diff
@@ -1,7 +1,8 @@
+
 [[ref-daemon-event-translate]]
 = Event Translator
 
-The event translator transforms and enhances the data from {page-component-title} events that other daemons generate.
+The event translator (ET) transforms and enhances the data from {page-component-title} events that other daemons generate.
```

Comment:
No need to include an abbreviation, since we don't use it anywhere else on the page.

Reply:
Suggestion to remove to make that change easier.
```suggestion
The event translator transforms and enhances the data from {page-component-title} events that other daemons generate.
```